src/argilla/server/daos/models/records.py

```diff
@@ -21,6 +21,7 @@
 from pydantic.generics import GenericModel
 
 from argilla import _messages
+from argilla._constants import PROTECTED_METADATA_FIELD_PREFIX
```

Comment:
It looks a bit dangerous to customize the prefix to identify protected metadata fields. Since is a constant value, I think it should be placed with the rest of the common constants.